src/ert/scheduler/lsf_driver.py

```diff
@@ -103,6 +103,21 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
     return {"jobs": data}
 
 
+def get_missing_jobids_from_bjobs_output(
+    jobs: dict[str, tuple[int, AnyJob, int]], bjobs_states: _Stat
+) -> set[str]:
+    current_timestamp = int(time.time())
+    expected_jobids = list()
+    for job_id, job_data in jobs.items():
+        submitted_timestamp = job_data[2]
+        if current_timestamp < submitted_timestamp + _POLL_PERIOD:
+            continue
+        expected_jobids.append(job_id)
```

Comment:
Can this be made shorter by a single list comprehension?

Reply:
I made it into a single set comprehension. A bit less readable due to less obvious tuple index for `submitted_timestamp`, but I think it should be fine.